### PR TITLE
Feature cucumber negatives for Account feature

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/test/resources/features/Account.feature [hunk 1]
```diff
@@ -7,22 +7,83 @@ Feature: Endpoint Account
...
+    When I use endpoint "sobjects/Account/" request to account with <apiName> and <value>
+    Then the response status code fail should be <status> to account
+    Examples:
+      | apiName      | value    | status|
+      | Nameeee      | Account1 | 400 |
+      | Description  | Account2 | 400 |
+      | null         | Account3 | 400 |
+
+
+
```
Comment (GustavoZHA) [inline src/test/resources/features/Account.feature]: Delete blank line

Comment (jpgonzales27) [reply src/test/resources/features/Account.feature]: Ok Gus, I will delete

File: src/test/resources/features/Account.feature [hunk 2]
```diff
@@ -7,22 +7,83 @@ Feature: Endpoint Account
...
 
+  @DeleteAccount
+  Scenario Outline: DELETE to account fail
+    Given I build "DELETE" request to account
+    When I use endpoint "sobjects/Account/{accountId}" request to account with "accountId" fail <badId>
+    And I execute the request account
+    Then the response status code fail should be <status> to account
+    Examples:
+      | badId |status|
+      | 12341415 | 404 |
```
Comment (GustavoZHA) [inline src/test/resources/features/Account.feature:60]: Organize the tables like line 73

File: src/test/resources/features/Account.feature [hunk 3]
```diff
@@ -7,22 +7,83 @@ Feature: Endpoint Account
...
+      | null | 404|
+
+  @PatchAccount
+  Scenario Outline: PATCH to account fail body
+    Given I build "PATCH" request to account
+    When I use endpoint "sobjects/Account/{accountId}" request to account with "accountId"
+    And I update account the <apiName> to <value>
+    Then the response status code fail should be <status> to account
+    Examples:
```
Comment (GustavoZHA) [inline src/test/resources/features/Account.feature:87]: Delete Examples extra

**Pull request merged**